allow use of any service description format, but recommend OpenAPI 3.0 or 3.1 (#226)
radiantearth/stac-api-spec: ogcapi-features/README.md

The following Link relations shall exist in the Landing Page (root).

| **rel**        | **href**       | **From**  | **Description**                  |
| -------------- | -------------- | --------- | -------------------------------- |
| `root`         | `/`            | STAC Core | The root URI                     |
| `self`         | `/`            | OAFeat    | Self reference, same as root URI |
| `conformance`  | `/conformance` | OAFeat    | Conformance URI                  |

> | `service-desc` | `/api`         | OAFeat    | The service description          |

Reviewer: ```suggestion
| `service-desc` | `/api`         | OAFeat    | The service description in a machine-readable format |
```

| `data`         | `/collections` | OAFeat    | List of Collections              |

The path for the `service-desc` endpoint is recommended to be `/api`, but may be another path. Recommended to be

> OpenAPI 3.0 or 3.1 with media types `application/vnd.oai.openapi` (YAML),

Reviewer: see above

`application/vnd.oai.openapi+json;version=3.0` (3.0 JSON), or `application/vnd.oai.openapi+json;version=3.1`
(3.1 JSON).

Additionally, a `service-doc` endpoint is recommended, but not required.

| **rel**       | **href**    | **From** | **Description**                                                                                                                                                                                                |
| ------------- | ----------- | -------- | -------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |

> | `service-doc` | `/api.html` | OAFeat   | An HTML service description.  Uses the `text/html` media type to refer to a human-consumable description of the service. The path for this endpoint is only recommended to be `/api`, but may be another path. |

Reviewer: see above

| Endpoint                                        | Returns                                                 | Description                                                                                                                                                             |
| ----------------------------------------------- | ------------------------------------------------------- | ----------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
| `/`                                             | [Catalog](../stac-spec/catalog-spec/README.md)          | Landing page, links to API capabilities                                                                                                                                 |

> | `/api`                                          | any                                                     | Returns an OpenAPI description of the service from the `service-desc` link `rel`. The path for this endpoint is only recommended to be `/api`, but may be another path. |

Reviewer: OpenAPI is wrong, it could be anything. Probably also an issue in several other places, did only realize it now...

```suggestion
| `/api`                                          | any                                                     | Returns a machine-readable API description of the service from the `service-desc` link `rel`. The path for this endpoint is only recommended to be `/api`, but may be another path. |
```
